Readme.md

```diff
@@ -44,28 +44,29 @@ default: false
 
 Whether to show [Brotli](https://www.wikiwand.com/en/Brotli) size or not
 
+#### showBeforeSizes
+type: `boolean`
+default: false
+
+Whether to show a comparison between the `output.file` file size as it was
+and as it is now being written. Useful to compare size of a new release to
+the previous one (though note that if you run Rollup multiple times, this
+info will be lost, except if still in your terminal history).
+
 #### format
 type : `object`
 
 default : {}
 
 See the options [here](https://github.com/avoidwork/filesize.js)
```

Comment:
Not related to this PR. But we should change the link to https://github.com/avoidwork/filesize.js#optional-settings so that the user lands at the correct place.

Reply:
Sure. I've rebased and changed this. Will look into `@babel/eslint-parser` next.

Reply:
When attempting a (Yarn) install, I get:

> An unexpected error occurred: "https://registry.npmjs.org/@babel%2feslint-parser: Not found".

I also see nothing at https://www.npmjs.com/package/@babel/eslint-parser and "package not found" on the npm badge at https://github.com/babel/babel/tree/master/eslint/babel-eslint-parser (and its `package.json` version set to `0.0.0`), so this apparently isn't yet ready for release.